config/locales/en.yml

```diff
@@ -2022,6 +2022,12 @@ en:
       ny_spouse_state_id:
         edit:
           title_html: Please provide information for <strong>your spouse’s</strong> state issued ID
+      pending_federal_return:
+        edit:
+          title_html: |
+            We’re sorry to keep you waiting. Your federal tax return must be accepted before you can file your state tax return.
```

Comment:
why not just put in <p> tags instead of having two </br>

Reply:
good question, no reason

Reply:
actually i guess the reason is that it's already in an h1 because it's the only text on the page